### DISCUSSION TITLE
Anybody hit with the swipe/pulldown and lost their updates ?

### DISCUSSION THREAD

**Discussion by davexnet** [bromite/bromite#1270] [Ideas]
I was wondering if it were possible to turn off this feature,
I've lost a few forum posts I was in the process of typing into the webpage,  
accidentally invoked the page refresh with a pulldown (I thought I was scrolling the page)
and the half-competed post disappeared before my eyes.

I've switched to Firefox to enter anything into a webpage, too easy to lose it in
Chromium/Bromite

**Comment by davexnet** [comment]
This thread explains the issue 
https://www.reddit.com/r/firefox/comments/q9x3hp/pull_down_to_refresh_on_android/

**Comment by csagan5** [comment]
This feature was there and apparently got removed: https://www.chromestory.com/2019/07/disable-pull-to-refresh-on-chrome-for-android/

The switch is still there though; we could probably restore the flag for it.

**Reply by ghost** [reply]
Was the flag ever restored?  I can't find it in the options for the latest bromite build.  I really like this browser, but when I'm scrolling up through YouTube comments, its annoying to have the page refresh by accident and then the video has to reload over again.

**Reply by n-buna404** [reply]
> Was the flag ever restored? I can't find it in the options for the latest bromite build. I really like this browser, but when I'm scrolling up through YouTube comments, its annoying to have the page refresh by accident and then the video has to reload over again.

Yes it was restored. go to chrome://flags/
search: pull
disable the pull-to-refresh-effect and then restart bromite twice for it take effect.

**Reply by ghost** [reply]
Awesome, thank you.  Would you consider putting it in the Options menu as an easy-to-toggle switch as well?

**Reply by n-buna404** [reply]
I am not the developer just a mere bromite user lurking around. You may submit feature request if u deems the switch important to considered.

**Comment by csagan5** [comment]
> I've switched to Firefox to enter anything into a webpage, too easy to lose it in
> Chromium/Bromite

Please refrain from these comments: if you like Bromite, that's great! If you like another browser, that's great as well! But no need to come and tell here on **Bromite Discussions**.

**Reply by davexnet** [reply]
I only mentioned it as a way to reiterate my point; I lost some in-progress forum posts because of the inadvertent refresh.
Perhaps I should not have mentioned the other browsers name.
I have both on my phone at all times and I switch back and forth
often having both open at the same time.  I like Bromite, otherwise I wouldn't have it installed.  Doesn't have to be excusive/or

It would be nice to see this flag back
Thanks

**Reply by davexnet** [reply]
Thanks very much for adding this flag.  Bromite is getting better and better, much appreciated
